soc/arm/atmel_sam0/common/soc_samr3x_radio_off.c

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2021 Argentum Systems Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/devicetree.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/init.h>
+
+/* When the radio is not in use, it's important that #CS is set high, to avoid
+ * unexpected behavior and increased current consumption... see Chapter 10 of
+ * DS70005356C. We also hold the radio in reset.
+ */
+static int radio_off_setup(void)
+{
+	int ret;
+	const struct gpio_dt_spec cs = GPIO_DT_SPEC_GET(DT_NODELABEL(sercom4), cs_gpios);
+	const struct gpio_dt_spec reset = GPIO_DT_SPEC_GET(DT_NODELABEL(lora), reset_gpios);
+
+	if (!gpio_is_ready(&cs) || !gpio_is_ready(&reset)) {
+		return -ENODEV;
+	}
+
+	ret = gpio_pin_configure_dt(&cs, GPIO_OUTPUT_INACTIVE);
+	if (ret < 0) {
+		return ret;
+	}
+
+	ret = gpio_pin_configure_dt(&reset, GPIO_OUTPUT_ACTIVE);
+	if (ret < 0) {
+		return ret;
+	}
```

Comment:
If you're returning on error, please assert reset before `CS`... (?)

Alternatively, attempt to set both, and _then_ return success/fail.